addons/l10n_id/i18n/id.po

```diff
@@ -0,0 +1,55 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_id
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 17.1alpha1+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-12-22 08:18+0000\n"
+"PO-Revision-Date: 2023-12-22 08:18+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_id
+#. odoo-python
+#: code:addons/l10n_id/models/res_bank.py:0
+#: code:addons/l10n_id/models/res_bank.py:0
+msgid ""
+"Communication with QRIS failed. QRIS returned with the following error: %s"
+msgstr "Komunikasi dengan QRIS gagal. QRIS kembali dengan error berikut: %s"
+
+#. module: l10n_id
+#. odoo-python
+#: code:addons/l10n_id/models/res_bank.py:0
+msgid "Could not establish a connection to the QRIS API."
+msgstr "Gagal membuat sambungan ke API"
+
```

Comment:
empty line. Moreover this file is in `i18n` and your remove pot in `i18n_extra`, you can move the pot here instead

Reply:
I think pot is not needed in l10n modules? As they're not used anyway

Thanks for the override

Reply:
Why do you say they are not used? We use .pot files when reading po
https://github.com/odoo/odoo/blob/17.0/odoo/tools/translate.py#L639-L651